### DISCUSSION TITLE
能介绍一下enjoy应用用途吗

### DISCUSSION THREAD

**Discussion by L-Super** [ZuodaoTech/everyone-can-use-english#199] [General]
README有提到enjoy应用，我不清楚和文章有什么关系，并且还需要一个mixin账号才能登录。文章中似乎也没提到enjoy，我不清楚它存在的意义，希望能告知一下

**Comment by Colorfulnia** [comment]
没有关系，一个是文章，一个是应用。一个repo里有两个东西（

**Comment by an-lee** [comment]
参考文档 https://1000h.org/enjoy-app/
